app/models/submission.rb

```diff
@@ -281,14 +281,14 @@ def self.get_satisfaction(satisfaction)
   end
 
   CSV_COLUMNS = [
-    'Response #', 'Source', 'Date', 'How Are You Testing', 'Zip', 'Census Tract', 'Census Block',
+    'Response #', 'Source', 'Date', 'How Are You Testing', 'Zip', 'Census Tract', 'Census Block', 'Latitude', 'Longitude', 'Accuracy',
     'Provider', 'How are you connected', 'Price Per Month', 'Advertised Download Speed',
     'Satisfaction Rating', 'Download Speed', 'Upload Speed', 'Advertised Price Per Mbps',
     'Actual Price Per Mbps', 'Ping'
   ]
 
   CSV_KEYS = [
-    :id, :source, :date, :testing_for, :zip_code, :census_code, :census_block, :provider, :connected_with,
+    :id, :source, :date, :testing_for, :zip_code, :census_code, :census_block, :latitude, :longitude, :accuracy, :provider, :provider, :connected_with,
```

Comment:
Would you remove the doubled `:provider` in here? That only needs to be included once.

Reply:
Thanks Noah. Silly mistake on my part. I fixed it just now.